### PR TITLE
REXControls.REXYGEN2.50.12.1471164bit version 2.50.12.14711

### PR DESCRIPTION
**Pull request by ch3sn3k**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108913)

### CODE REVIEW

**Comment by ch3sn3k** [comment]
@microsoft-github-policy-service agree company="REX Controls s.r.o."

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
This package installs normally in a Windows 11 VM, which might be due to the dependency being bundled with the newer OS.

**Comment by BrandonWanHuanSheng** [comment]
> This package installs normally in a Windows 11 VM, which might be due to the dependency being bundled with the newer OS.

Yes, it is true. I already found Windows 11 Feature Updates contain this Microsoft-Windows-EditionPack-Core-Package.ESD\amd64_netfx4-vcruntime140_clr_dll_31bf3856ad364e35_4.0.15912.0_none_746f6f2d2bca9bfc\. So windows 11 use that thing from C:\Windows\servicing\LCU\Package_for_RollupFix~31bf3856ad364e35~amd64~~10.0.22621.1

**Comment by BrandonWanHuanSheng** [comment]
Screenshot (247)

**Comment by ch3sn3k** [comment]
Dear @BrandonWanHuanSheng ,
I did not understand if I should somehow include the library `vcruntime140_1.dll` in the installer or if it was a mistake on the test VM end? Our users normally install the REXYGEN on both Windows 10 and 11. Also, the installer was compatible with Windows 7.
Can you help me what should I do?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/r/REXControls/REXYGEN2/50/12/1471164bit/2.50.12.14711/REXControls.REXYGEN2.50.12.1471164bit.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: REXControls.REXYGEN2.50.12.1471164bit
+PackageVersion: 2.50.12.14711
+Installers:
```
Comment (stephengillie) [inline manifests/r/REXControls/REXYGEN2/50/12/1471164bit/2.50.12.14711/REXControls.REXYGEN2.50.12.1471164bit.installer.yaml:6]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Comment by stephengillie** [comment]
With the dependency preinstalled, the package installs and launches normally. 
image

**Comment by ch3sn3k** [comment]
Hello @stephengillie ,
I added the dependency to the `yaml` file as you proposed, but unfortunately, the automatic build failed. Is there anything else I should do to move forward? I would appreciate any help.
Thank you in advanced

**Comment by stephengillie** [comment]
> Hello @stephengillie , I added the dependency to the `yaml` file as you proposed, but unfortunately, the automatic build failed. Is there anything else I should do to move forward? I would appreciate any help. Thank you in advanced

Hi @ch3sn3k,

Unfortunately, new packages with dependencies are being held for dependency support. If it's okay with you, we'll keep this PR with the others requiring dependencies until we're ready for them to merge.

**Comment by ch3sn3k** [comment]
@stephengillie  Thank you for your help. I will prepare a new installer with all dependencies included and create a new PR.

**Pull request closed without merging**